pdfjs.config

```diff
@@ -1,6 +1,6 @@
 {
   "betaVersion": "1.2.109",
   "stableVersion": "1.1.469",
-  "baseVersion": "2096a2a94a1f4971a15317bd75d4ec08c7f78f7f",
-  "versionPrefix": "1.2."
+  "baseVersion": "194994a2898a6d01c434d6d3270357799f3ac820",
```

Comment:
The version shall start at the moment we break API. Since we did not release 1.3 yet it's okay to provide fixes and improvements for 1.3 changes.